addons/mass_mailing/tests/test_mailing_ui.py

```diff
@@ -20,4 +20,4 @@ def test_01_mass_mailing_editor_tour(self):
         # for email client compatibility should be saved in body_html. This
         # ensures both fields have different values.
         self.assertEqual(mail.body_arch, '<p><br></p>')
-        self.assertEqual(mail.body_html, '<p style="margin:0px 0 14px 0;box-sizing:border-box;"><br></p>')
+        self.assertEqual(mail.body_html, '<p style="margin:0px 0 14px 0;"><br></p>')
```

Comment:
Since this commit is now separated from the original PR, I think it would be better to have the xml fix commit hash in this commit's message.